--- ChangeLog	2025-01-15 18:41:57.097331308 -0500
+++ ChangeLog-spell-corrected	2025-01-15 18:43:22.119360475 -0500
@@ -4,7 +4,7 @@

 2024-12-09  rocky <rocky@gnu.org>

-	* pyficache/version.py: Bump version due to API incompatability
+	* pyficache/version.py: Bump version due to API incompatibility

 2024-12-06  rocky <rocky@gnu.org>

@@ -55,7 +55,7 @@

 2024-10-16  rocky <rocky@gnu.org>

-	* test/test_pyficache.py: Reduce use of Python verison differences
+	* test/test_pyficache.py: Reduce use of Python version differences
 	in test

 2024-10-16  rocky <rocky@gnu.org>
@@ -506,6 +506,6 @@
 2021-11-28  rocky <rocky@gnu.org>

 	* admin-tools/setup-python-2.4.sh, admin-tools/setup-python-3.1.sh,
-	admin-tools/setup-python-3.3.sh: vresion hacking
+	admin-tools/setup-python-3.3.sh: version hacking

 2021-11-28  rocky <rocky@gnu.org>

@@ -765,7 +765,7 @@
 2020-05-21  rocky <rocky@gnu.org>

 	* requirements.txt: A more accurate requirements.txt CircleCI seems to need this in its default configuration.  Until I
-	figure out how to aviod this, we'll duplicate the config.
+	figure out how to avoid this, we'll duplicate the config.

 2020-05-21  rocky <rocky@gnu.org>

@@ -895,7 +895,7 @@

 2018-01-26  rocky <rocky@gnu.org>

-	* pyficache.py: Add reverse option on unmap_file_line...  To allow retriving line number in the other direction.
+	* pyficache.py: Add reverse option on unmap_file_line...  To allow retrieving line number in the other direction.

 2018-01-25  rocky <rocky@gnu.org>

@@ -1012,7 +1012,7 @@
 2015-12-26  rocky <rocky@gnu.org>

 	* Makefile, pyficache.py: Fix bug in setting pygments style, Line
-	remapping stuff is stil borked though.
+	remapping stuff is still borked though.

 2015-12-07  rocky <rocky@gnu.org>

@@ -1267,6 +1267,6 @@

 2012-12-30  rocky.bernstein <rocky.bernstein@928b09c6-f64b-11dd-aa46-671127399dce>

-	* pyficache.py: Use newer interface more agressively. Fix
+	* pyficache.py: Use newer interface more aggressively. Fix
 	uninitialized dict key bug.  git-svn-id: https://pyficache.googlecode.com/svn/trunk@32
 	928b09c6-f64b-11dd-aa46-671127399dce
